lisa/tools/ntttcp.py

```diff
@@ -171,7 +171,7 @@ def get_throughput(self, stdout: str) -> str:
     def run_as_server_async(
         self,
         nic_name: str,
-        run_time_seconds: int = 10,
+        run_time_seconds: int = 300,
```

Comment:
Keep the default value to 10 seconds. It's used not for perf testing, also functional. Extend it will make test cases take unnecessary longer time.

Reply:
Changed back to 10s